packages/react-reconciler/src/ReactFiberWorkLoop.new.js

```diff
@@ -510,6 +513,13 @@ function requestRetryLane(fiber: Fiber) {
   return claimNextRetryLane();
 }
 
+export function getSpawnedLane(): Lane {
+  if (workInProgressRootSpawnedLane === NoLane) {
+    workInProgressRootSpawnedLane = claimNextTransitionLane();
+  }
+  return workInProgressRootSpawnedLane;
```

Comment:
Do they need to be the same one? Is that even desirable?

It’s unfortunate that we need another thread-local field. It’s getting to be a lot.

Reply:
Changed it to grab a new one each time